docs/GettingStarted.rst

```diff
@@ -300,6 +300,485 @@ redirected to the :ref:`Social Intranet homepage <PLFHomepage>`. Now, you can st
        - \*:/developers
 
 	* If the server stops before your account setup data is submitted, the Account Setup screen will appear at your next startup.
+
+  .. _DWHomepage:
+
+==========================
+Digital Workplace Homepage
+==========================
+
+This section introduces you to the Digital Workplace homepage, ie **Snapshot**. Besides,
+you will further learn about the following topics:
+   
+ * :ref:`Digital Workplace applications <Intranet-Gadgets>`
+   Introduction to applications that come with the Digital Workplace
+   homepage.
+   
+ * :ref:`Changing the UI language <Change-Language>`
+   Instructions on how to change the UI language of eXo Platform.
+
+After signing in your account successfully, you are redirected to the
+**Digital Workplace** homepage, the starting point for exploring eXo Platform.
+This homepage provides you a perfect overview of all social and
+collaboration activities available in eXo Platform.
+
+|image4|
+
+There are 3 main divisions in the **Digital Workplace** homepage:
+
+**Navigations**
+
+
+-  *Top navigation* |image5|: Take common actions via the following
```

Comment:
Top navigation is not specific to the homepage. It’everywhere in digital workplace and even in legacy intranet site’ Shouldn’t we describe it in a dedicated chapter  rather than here inside qhomepage description?